plugins/upload/src/lib.rs

```diff
@@ -66,10 +66,14 @@ async fn download(
     file_path: &str,
     headers: HashMap<String, String>,
     on_progress: Channel,
+    body: &str,
 ) -> Result<()> {
     let client = reqwest::Client::new();
-
-    let mut request = client.get(url);
+    let mut request = if !body.is_empty() {
+        client.post(url).body(body.to_owned())
```

Comment:
If we need an owned body var then you can just use `body: String` as the function argument.